src/Controller/EventAttendanceController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @file
+ */
+
+namespace Drupal\popapi\Controller;
+
+use Drupal\popapi\Controller\EventNodeController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class EventAttendanceController
+ * @package Drupal\popapi\Controller
+ */
+class EventAttendanceController {
+
+  /**
+   * @param Request $request
+   * @return JsonResponse
+   */
+  public function insertAttendee(Request $request) {
+    $data = json_decode($request->getContent(), TRUE);
+    $nid = $this->getEventNodeId($data['event_id']);
+
+    if ($this->addAttendeeCount($nid)) {
+      return new JsonResponse($this->getStatus(1));
+    } else {
+      return new JsonResponse($this->getStatus(0));
+    }
+
+  }
+
+  /**
+   * @param $event_id
+   * @return mixed
+   */
+  private function getEventNodeId($event_id) {
+    $load_id = EventNodeController::loadEventAttendeeCount($event_id);
+
+    foreach ($load_id as $id) {
+      return $id;
+    }
+  }
+
+  /**
+   * @param $nid
+   * @return bool
+   */
+  private function addAttendeeCount($nid) {
```

Comment:
incrementAttendee() returns a bool so we can just have

`return EventNodeController::incrementAttendee($nid);`

However, cant incrementAttendee just do the job? It seems addAttendeeCount is only returning whatever incrementAttendee returns